src/json.cpp

```diff
@@ -214,6 +214,21 @@ bool JsonObject::get_bool(const std::string &name, const bool fallback)
     return jsin->get_bool();
 }
 
+bool JsonObject::get_bool_or_flag( const std::string &name, const std::string &flag, const bool fallback, const std::string &flags_node )
+{
+    const std::set<std::string> flags = get_tags( flags_node );
+    bool value = fallback;
+    if( has_bool( name ) ) {
+        std::stringstream err;
+        err << "JsonObject contains legacy node `" << name << "`.  Consider replacing it with `" << flag << "` flag in `" << flags_node << "` node.";
+        JsonObject::throw_error( err.str() );
```

Comment:
`throw_error` doesn't return normally, writing any code after it is pointless (`value = ...` will *never* be run).